webpack.config.js

```diff
@@ -90,16 +120,26 @@ module.exports = {
 		],
 	},
 	mode,
-	entry: gutenbergPackages.reduce( ( memo, packageName ) => {
-		const name = camelCaseDash( packageName );
-		memo[ name ] = `./packages/${ packageName }`;
-		return memo;
-	}, {} ),
+	entry: createEntrypoints(),
 	output: {
 		devtoolNamespace: 'wp',
-		filename: './build/[basename]/index.js',
+		filename: ( data ) => {
+			const { chunk } = data;
+			const { entryModule } = chunk;
+			const { rawRequest } = entryModule;
+
+			/*
+			 * If the file being built is a Core Block's frontend file,
+			 * we build it in the block's directory.
+			 */
+			if ( rawRequest && rawRequest.includes( '/frontend.js' ) ) {
+				return `./build/block-library/blocks/[name]/frontend.js`;
```

Comment:
That's nice, it's going to be located next to `block.json` in the build folder so it should be a simple reference once we introduce `frontend` field in `block.json`. I like it 👍🏻